toc/rfc/rfc-draft-pcap-bosh.md

```diff
@@ -0,0 +1,59 @@
+# Meta
+[meta]: #meta
+- Name: Integrate pcap feature into BOSH
+- Start Date: 2023-07-06
+- Author(s): @domdom82 @maxmoehl
```

Comment:
```suggestion
- Author(s): [@domdom82](https://github.com/domdom82)  [@maxmoehl](https://github.com/maxmoehl)
```

Reply:
I don't think those are supposed to be links. Userhandles should be fine. (double check other RFCs, e.g. [here](https://github.com/cloudfoundry/community/blob/main/toc/rfc/rfc-0010-stemcell-cleanup.md) )